web/themes/custom/cloudy/src/css/_overrides.scss

```diff
@@ -258,3 +258,13 @@ drupal-entity { display: inline; }
   clip: auto !important;
   white-space: normal !important;
 }
+
+.visually-hidden { 
+  position: absolute !important;
+  height: 1px; 
+  width: 1px;
+  overflow: hidden;
+  clip: rect(1px 1px 1px 1px); /* IE6, IE7 */
+  clip: rect(1px, 1px, 1px, 1px);
+  white-space: nowrap; /* added line */
+}
```

Comment:
I looked further into this. This is a chunk of styles that's coming from a module in core. Let's remove it from here, and change the class on the twig component to sr-only.